### ISSUE TITLE
Different weather radars

### ISSUE THREAD

**Issue by JXNXH28** [flybywiresim/aircraft#8459]
### Aircraft Version

Stable

### Description

I'd like to have the option to choose between the different weather radars like the RDR-4000, RDR-4B and the current WXR-2100

### References (optional)

https://safetyfirst.airbus.com/optimum-use-of-weather-radar/

### Discord Username (optional)

jonah9224

**Comment by tracernz** [comment]
There is no possibility to build any functional weather radar at the moment so it seems a little irrelevant for now.

**Comment by JXNXH28** [comment]
I totally agree. I'm just thinking for when there is a working weather radar, it would be nice to have.

**Comment by Mike737377** [comment]
@JXNXH28 I've been using [this wx radar mod](https://flightsim.to/file/57681/weather-radar-mod-for-fbw-a320-neo-development-version) as a workaround for awhile now. It's not perfect, but it's better than nothing.
